addons/website/static/src/scss/website.scss

```diff
@@ -450,10 +450,15 @@ table.table_desc tr td {
 
 // Three columns
 
-.s_three_columns {
+.s_three_columns, .s_comparisons {
     .align-items-stretch .card {
         height: 100%;
     }
```

Comment:
Maybe it should be a common style instead ? Maybe directly in bootstrap_review.scss ?

Reply:
This was only a way to group the `overflow: hidden` for both types of cards, the comparison snippet doesn't use align-items-stretch. It also doesn't seem like a great rule in the first place: if you insert a card inner content snippet inside of the three columns snippet it breaks the layout. This rule should either have higher specificity or be replaced by h-100 classes on each individual card directly in the snippet's XML.

Reply:
Mmh, the bug is rather that we can drop a .card in a .card but in any case I think that the rule `.row > div > .card { height: 100%; }` would make sense as it would make the card fit the columns for all snippets when possible.

Reply:
> Mmh, the bug is rather that we can drop a .card in a .card

Considering that tabs are implemented using a card, I'm not sure we want to remove that completely.

I'll go with your CSS rule, it seems like the most sensible solution.